smarts/core/road_map.py

```diff
@@ -55,14 +55,6 @@ def bounding_box(self) -> BoundingBox:
         # may return None to indicate the map is unbounded.
         raise NotImplementedError()
 
-    @property
-    def xy_offset(self) -> Tuple[float, float]:
-        """optional: this is the amount that external coordinates must
-        be shifted in order to have the coordinate system (bounding_box)
-        start at the origin.  Will be (0. 0) for most maps."""
-        # TAI:  get rid of this, fix traffic_history_provider instead
-        return (0, 0)
-
```

Comment:
While I'm happy to see this go, I'm a bit nervous by something that Jun said in a recent tech meeting about keeping things as close to the data as possible.  I think we should still go ahead with this, but also be prepared to bring this back in the future if necessary.